lib/reporters/json.js

```diff
@@ -83,7 +83,8 @@ function clean (test) {
     fullTitle: test.fullTitle(),
     duration: test.duration,
     currentRetry: test.currentRetry(),
-    err: cleanCycles(err)
+    err: cleanCycles(err),
+    reason: test.reason
```

Comment:
1.  Can we suppress addition of the `reason` property in the case of a **non-pending** test?
1.  Can we coerce `reason` to a `String` (e.g. `reason: String(test.reason)`)